src/features/island/collectibles/RemoveCropMachineModal.tsx

```diff
@@ -32,7 +32,7 @@ export const RemoveCropMachineModal: React.FC<Props> = ({
         </div>
 
         <Button onClick={onRemove} className="mt-2">
-          {t("removeCropMachine.removeSeeds")}
+          {t("removeCropMachine.removeOil")}
```

Comment:
<img width="554" alt="Screenshot 2024-06-03 at 11 53 52 AM" src="https://github.com/sunflower-land/sunflower-land/assets/11745561/1ac8c78b-01d6-42d9-b862-6340cc77109c">

Since this action will remove the entire building, consider just naming it "Remove"